.mailmap

```diff
@@ -1,71 +1,95 @@
 # https://www.kernel.org/pub/software/scm/git/docs/git-shortlog.html#_mapping_authors
-Jarkko Hietaniemi <jhi@iki.fi> <jhi@alpha.hut.fi>
+Aaron Crane <arc@cpan.org> <perl@aaroncrane.co.uk>
+Abhijit Menon-Sen <ams@toroid.org> <ams@toroid.org>
+Andy Dougherty <doughera@lafayette.edu> <doughera@lafayette.edu>
+Andy Dougherty <doughera@lafayette.edu> Andy Dougherty <doughera.lafayette.edu>
+Andy Dougherty <doughera@lafayette.edu> Andy Dougherty <doughera@fractal.phys.lafayette.edu>
+Andy Dougherty <doughera@lafayette.edu> Andy Dougherty <doughera@lafcol.lafayette.edu>
+Andy Dougherty <doughera@lafayette.edu> Andy Dougherty <doughera@newton.phys.lafayette.edu>
+Audrey Tang <cpan@audreyt.org> Autrijus Tang <unknown>
+Audrey Tang <cpan@audreyt.org> autrijus@ossf.iis.sinica.edu.tw <autrijus@ossf.iis.sinica.edu.tw>
+Ævar Arnfjörð Bjarmason <avar@cpan.org> Ævar Arnfjörð Bjarmason <avarab@gmail.com>
+Chad Granum <exodist7@gmail.com> <exodist7@gmail.com>
+Chip Salzenberg <chip@atlantic.net> Chip <chip@pobox.com>
+Chip Salzenberg <chip@atlantic.net> Chip Salzenberg <chip@ci005.sv2.upperbeyond.com>
+Chip Salzenberg <chip@atlantic.net> Chip Salzenberg <chip@perl.com>
+Chip Salzenberg <chip@atlantic.net> Chip Salzenberg <chip@pobox.com>
+Chip Salzenberg <chip@atlantic.net> Chip Salzenberg <salzench@dun.nielsen.com>
+Chip Salzenberg <chip@atlantic.net> Chip Salzenberg <salzench@nielsenmedia.com>
+Chris 'BinGOs' Williams <chris@bingosnet.co.uk> <chris@bingosnet.co.uk>
+Chris 'BinGOs' Williams <chris@bingosnet.co.uk> Chris BinGOs Williams <chris@bingosnet.co.uk>
+Chris 'BinGOs' Williams <chris@bingosnet.co.uk> Chris Williams <chris@bingosnet.co.uk>
```

Comment:
There's a lot of entries like this in the file that are redundant.  A single entry like:
```
Chris 'BinGOs' Williams <chris@bingosnet.co.uk> <chris@bingosnet.co.uk>
```
will change the name and email of anything matching the email `chris@bingosnet.co.uk`.

Reply:
blame me for that, not rjbs. he's just sorting the file.